guides/v2.2/magento-functional-testing-framework/release-2/update.md

```diff
@@ -15,27 +15,41 @@ Magento tests and the framework are stored in different repositories.
 
 Magento tests are stored in the same repository as the Magento code base. When you pull changes in the Magento code, you're potentially pulling corresponding tests as well.
 
-The MFTF is installed separately as a dependency using Composer. When pulling the latest Magento code, update the corresponding Composer dependencies in the `magento2/dev/tests/acceptance` directory. This ensures that the MFTF is up to date.
+The MFTF is installed separately as a dependency using Composer. When pulling the latest Magento code, update the corresponding Composer dependencies in the `magento2` root directory. This ensures that the MFTF is up to date.
 
-## Update the MFTF
+## Update the MFTF to 2.3.0
 
 To update the MFTF (via a command line interface):
 
 1. When you update Magento, verify that the Magento [WYSIWYG settings](getting-started.html#wysiwyg-settings) and [Security settings](getting-started.html#security-settings) are set appropriately.
-1. Go to the `magento2/dev/tests/acceptance` directory:
+1. Starting at the `magento2` root directory remove the vendor folder:
 
 	```bash
-	cd dev/tests/acceptance
+	rm -rf vendor/
```

Comment:
What does this accomplish?  The composer.json file references a specific Magento version (eg 2.2.5).  Deleting vendor and then re-running `composer update` will still result in the same version of Magento.